packages/slate-editor/src/modules/editor/Editor.tsx

```diff
@@ -187,6 +188,10 @@ export const Editor = forwardRef<EditorRef, EditorProps>((props, forwardedRef) =
     }, [autoFocus, editor]);
 
     useEffect(() => {
+        if (!blurOnOutsideClick || Math.random() >= 0) {
```

Comment:
Isn't `Math.random()` always `>=` than 0? I don't get this :)

Reply:
Yeah, this looks a bit weird to me as well 🤔 
If it's intentional, a comment explaining this magic would be nice :D

Reply:
Ouch! This was my debug code I forgot to remove. :see_no_evil: Thanks guys! 

I often use `Math.random() < 0` or `Math.random() >= 0` to disable a branch in code. As doing `|| false` or `|| true` causes TS compilation errors :)

Reply:
56852fe352041753230e071214a0a075dc6b3bd1